bin/console

```diff
@@ -4,7 +4,4 @@ require "bundler/setup"
 require "browser_crawler"
 
 # You can add fixtures and/or initialization code here to make experimenting
-# with your gem easier. You can also use a different console, if you like.
-
-require "pry"
-Pry.start
```

Comment:
what is the reason of removing the call to console with pry?

Reply:
I reverted this changes